site/src/components/BlogPosts.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import DesigningMannequin from '../img/blog-posts/designing-mannequin.png'
+import ComponentTheming from '../img/blog-posts/component-theming.png'
+import IntroducingMannequin from '../img/blog-posts/introducing-mannequin.jpg'
+import './BlogPosts.scss'
+
+export default function BlogPosts() {
+  return (
+    <div className="BlogPosts">
+
+      <h2>Blog Posts</h2>
+
+      <ul>
+
+        <li>
+          <a href="https://lastcallmedia.com/blog/Designing-Mannequin-1.0" title="Designing Mannequin 1.0">
+            <img src={DesigningMannequin} alt="Designing Mannequin 1.0"/>
+            <span className="BlogPosts__tag">Design</span>
+            <span className="BlogPosts__title">Designing Mannequin 1.0</span>
+          </a>
+        </li>
+
+        <li>
+          <a href="https://lastcallmedia.com/blog/why-component-theming" title="Why we’re moving to component theming">
+            <img src={ComponentTheming} alt="Why we’re moving to component theming"/>
+            <span className="BlogPosts__tag">Coding + Development</span>
+            <span className="BlogPosts__title">Why we’re moving to component theming</span>
```

Comment:
It's kind of weird that we're mixing BEM style class names with the current style.  I'm not against it, but if you feel it's better, I think we should move toward this style in all future work on this project.  I'd rather not mix styles unless we're in the process of transitioning from one to the other.

Reply:
I was actually thinking that we should transition to it. I like the idea of having a clear process for making the markup that always provides and hooks we might need, rather than adding classNames at the moment we realize that we need a hook.

But if you have any other preferred naming technique, I'm down for that too. This is kind of my first attempt at BEM.

Reply:
I'm having trouble seeing the benefit for this project.  We already scope all of our styles such that elements (`.tag`) only have style within a block (`.BlogPosts`).  BEM syntax really only makes sense to me if we're using single level selectors (`.BlogPosts__tag`, not `.BlogPosts .BlogPosts__tag`).  I do think there's a specificity/performance argument to be made here, but I think it's actually worse for clarity and semantics when we're already nesting selectors.

Can you elaborate why you feel this makes the markup writing process clearer?

Reply:
No… I think you'r right. Changing it.

Reply:
Done.